x/genutil/client/cli/validate_genesis.go

```diff
@@ -41,7 +42,7 @@ func ValidateGenesisCmd(mbm module.BasicManager, txEncCfg client.TxEncodingConfi
 			}
 
 			var genState map[string]json.RawMessage
-			if err = cdc.UnmarshalJSON(genDoc.AppState, &genState); err != nil {
+			if err = legacyAmino.UnmarshalJSON(genDoc.AppState, &genState); err != nil {
```

Comment:
I think here we need to use encoding/json, right?

Reply:
yes I believe you're right, in these cases `encoding/json` should be sufficient.